src/components/icon/_icon-btn.less

```diff
@@ -1,6 +1,13 @@
 .icon-btn {
   color: @black;
 
+  [data-theme="dark"] &,
+  &.dark-mode {
+    & {
```

Comment:
This & is not necessary. You can remove this block and just put the rule `color: @white;`

Reply:
Thank @shahednasser. I will fix it later this day and let you know

Reply:
Hi, @shahednasser suggested fix was applied